Fx147 devtools edit pseudo element selector (#42316)
mdn/content: files/en-us/mozilla/firefox/releases/147/index.md

## Changes for web developers

### Developer Tools

> - When [pseudo-element](/en-US/docs/Web/CSS/Reference/Selectors/Pseudo-elements) nodes are selected in the HTML pane of the Inspector — for example, those representing generated content created by {{cssxref("::before")}} and {{cssxref("::after")}} rules — you can now edit the selectors of the corresponding rules in the Rule view.

Reviewer: - I referred to https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/ui_tour/index.html for the UI terminology and suggesting CSS pane instead of Rule view.
- I guess the pseudo-element itself can be selected in the HTML pane but if using the Inspector picker on the web page, then in that case you'd be selecting the content generated by the pseudo element.
```suggestion
- When you select a [pseudo-element](/en-US/docs/Web/CSS/Reference/Selectors/Pseudo-elements) (such as {{cssxref("::before")}} or {{cssxref("::after")}}) in the HTML pane of the Inspector – or select its generated content on the web page – you can now edit the selector of the corresponding rule in the CSS pane.
```
- Alternatively, we could skip "- or select its generated content on the web page -"

Author: Yeah, that makes sense to me. I've edited your suggestion to remove that part, and will now commit it.
